src/analysis/retail/monk/mistweaver/modules/spells/LifeCocoon.tsx

```diff
@@ -39,22 +33,6 @@ class LifeCocoon extends Analyzer {
       this.healing += calculateEffectiveHealing(event, LIFE_COCOON_HEALING_BOOST);
     }
   }
-
```

Comment:
Instead of removing/changing anything inside the module right now  just remove the entry from the Mistweaver CombatLogParser.ts file and it won't load